tests/test_novoCaller.py

```diff
@@ -14,6 +15,66 @@
 # this tests may fail because of differences in float handling
 # e.g. novoCaller=1.4030838153054451e-05 Vs novoCaller=1.403083815305445e-05
 # depending on the machine
+
+def assert_(tfile, ofile, test=False):
+    ''' '''
```

Comment:
Doc strings should use doublequotes not single quotes. Linters like `flake8` will flag this. It's good to fix it early to save pain later.